getline.ts/src/example.ts

```diff
@@ -3,10 +3,32 @@ import {BigNumber} from 'bignumber.js';
 
 import { Client, Loan, LoanState } from './index';
 
+function delay(ms: number): Promise<void> {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
 let main = async() => {
     let c = new Client("https://0.api.getline.in", "4");
     let user = await c.currentUser();
     console.log("user: " + user.ascii);
+
+    const testToken = c.testToken;
+    const balance = await testToken.humanize(await testToken.balanceOf(user));
+    console.log(`${await testToken.name()} (${await testToken.symbol()}) balance: ${balance.toString()}`)
+    if (balance.eq(0)) {
+        const printValue = await testToken.humanize(await testToken.printValue())
+        console.log(`Printing ${printValue.toString()} ${await testToken.symbol()}...`)
+        await testToken.print(user)
+
+        let newBalance = new BigNumber(balance);
+        while (newBalance.eq(balance)) {
```

Comment:
The polling loop shouldn't be needed at all, force update from blockchain should fix it